src/modules/CRUD/components/Details/PropertyItem/Edit/Edit.js

```diff
@@ -93,24 +105,35 @@ const Edit = ({
 
     setDefaultValue(propValue);
 
-    const savedFeatures = await saveFeature(
+    const isDeleting = isFeatureToDelete(isGeom, propValue, method);
+    const saveOrDeleteFeature = isDeleting
+      ? deleteFeature
+      : saveFeature;
+
+    const savedFeatures = await saveOrDeleteFeature(
       featureEndpoint,
       endpoint,
       body,
       method,
     );
 
     if (!isMounted.current) return;
-
-    setLoading(false);
-    if (savedFeatures.feature) {
+    if (!savedFeatures.error) {
+      getSettings(settingsEndpoint);
+      if (isDeleting) {
+        await fetchFeature(featureEndpoint, id);
```

Comment:
You need another mounted guard after this await I think.

Reply:
Done 👍